programs/ssl/ssl_client2.c

```diff
@@ -389,6 +397,30 @@ int main( void )
 #define USAGE_SERIALIZATION ""
 #endif
 
+#if defined(MBEDTLS_ZERO_RTT)
+#define USAGE_EARLY_DATA \
+    "    early_data=%%s        default: (library default: disabled)\n"      \
+    "                        options: disabled, enabled\n"
+#else
+#define USAGE_EARLY_DATA ""
+#endif /* MBEDTLS_ZERO_RTT */
+
+#if defined(MBEDTLS_ECP_C)
+#define USAGE_NAMED_GROUP \
+    "    named_groups=%%s    default: secp256r1, secp384r1\n"      \
+    "                        options: secp256r1, secp384r1, secp521r1, all\n"
+#else
+#define USAGE_NAMED_GROUP ""
+#endif /* MBEDTLS_ECP_C */
+
+#if defined(MBEDTLS_ECP_C)
+#define USAGE_KEY_SHARE_NAMED_GROUPS \
+    "    key_share_named_groups=%%s    default: secp256r1, secp384r1\n"      \
```

Comment:
Minor: This should likely be guarded by MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL, too.